### PR TITLE
Seltzer update

### PR DESCRIPTION
**Pull request by AHolmen**
#### Summary
Content "more interesting description of both HB seltzer and the commercial one. Removes vines in recipe."

### CODE REVIEW

**Comment by Maleclypse** [comment]
You could add variants to the commercial versions.

File: data/json/recipes/food/brewing.json [hunk 1]
```diff
@@ -228,15 +228,9 @@
     "tools": [ [ [ "surface_heat", 6, "LIST" ] ] ],
     "components": [
       [ [ "water", 10 ], [ "water_clean", 10 ] ],
-      [ [ "sugar", 2 ] ],
+      [ [ "sugar", 2 ], [ "artificial_sweetener", 2 ] ],
```
Comment (TheShadowFerret) [inline data/json/recipes/food/brewing.json:231]: Isn't this sugar supposed to be a substrate for the yeast 🤔

Comment (AHolmen) [reply data/json/recipes/food/brewing.json:231]: Yes, you're correct on both points.

**Comment by AHolmen** [comment]
> You could add variants to the commercial versions.

That is something worth considering!

File: data/json/recipes/food/brewing.json (hunk 1 above)
Comment (Fris0uman) [reply data/json/recipes/food/brewing.json]: I removed it

**Pull request merged**